muzzle/build.gradle.kts

```diff
@@ -0,0 +1,61 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
+}
+
+group = "io.opentelemetry.javaagent"
+
+//val instrumentationMuzzle by configurations.creating {
+//  isCanBeConsumed = true
+//  isCanBeResolved = false
+//  extendsFrom(configurations.implementation.get())
+//}
+
+dependencies {
```

Comment:
This module is just for `InstrumentationClassPredicate`, right? Can we put it in the `extension-api` module instead?

Reply:
For now. When I extract MuzzleCheck it may host more. And I am not sure about putting them into client-facing module.

Reply:
Hmm, right, the muzzle check plugin will bring a lot of shared stuff...